### PR TITLE
add to kb what happens during preconf

### PR DESCRIPTION
**Pull request by offgridauthor**
Added to knowledgebase a high level explainer for what happens during an execution preconf, and related SVG

### CODE REVIEW

File: knowledge-base/what-happens-during-an-execution-preconf.mdx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+- You might pay gas for failed transactions.
+- Your bundle may not get included, which is especially problematic for single-shot opportunities in DeFi. Some arbitrage opportunities only exist for a very brief window, and missing this window means losing the entire opportunity.
+
+## Using an Execution Preconfirmation
+
+<img height="200" src="/images/preconf-process.svg" />
+
+Instead of blindly sending your transaction, you can use an execution preconf. Here's how:
+
+**Note**: In the near future, you'll be able to add your transaction payload directly to your bid, which streamlines the process even further.
```
Comment (techgangboss) [inline knowledge-base/what-happens-during-an-execution-preconf.mdx:32]: add as footnote to drop maybe, as this feature will be shortly available

File: knowledge-base/what-happens-during-an-execution-preconf.mdx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+1. Setting up your mev-commit bidder node
+
+   - Follow the Quickstart guide to set up your mev-commit bidder node and fund your account.
```
Comment (techgangboss) [inline knowledge-base/what-happens-during-an-execution-preconf.mdx:36]: let's add links to docs, there's an easy way to link to pages within mintlify it's / something directory

File: knowledge-base/what-happens-during-an-execution-preconf.mdx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+2. Prepare your transaction
+
+   - Identify your MEV opportunity. Craft your bundle and send the signed transaction to block builders as per usual.
+
+3. Submit a preconf bid
+
+   - Create and submit a bid for the transaction from your mev-commit bidder node (following the bid structure here), supplying the hash of the transaction you just sent.
```
Comment (techgangboss) [inline knowledge-base/what-happens-during-an-execution-preconf.mdx:44]: says "here" but no link?

File: knowledge-base/what-happens-during-an-execution-preconf.mdx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+## Using an Execution Preconfirmation
+
+<img height="200" src="/images/preconf-process.svg" />
+
+Instead of blindly sending your transaction, you can use an execution preconf. Here's how:
+
+**Note**: In the near future, you'll be able to add your transaction payload directly to your bid, which streamlines the process even further.
+
+1. Setting up your mev-commit bidder node
```
Comment (techgangboss) [inline knowledge-base/what-happens-during-an-execution-preconf.mdx:34]: you will want to use mintlify features like the steps here to make it look better: https://mintlify.com/docs/content/components/steps#steps

File: knowledge-base/what-happens-during-an-execution-preconf.mdx [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+   - Follow the Quickstart guide to set up your mev-commit bidder node and fund your account.
+
+2. Prepare your transaction
+
+   - Identify your MEV opportunity. Craft your bundle and send the signed transaction to block builders as per usual.
```
Comment (techgangboss) [inline knowledge-base/what-happens-during-an-execution-preconf.mdx:40]: lowercase mev preferred :)

**Review by techgangboss — CHANGES_REQUESTED** [review]
left some comments to be addressed

File: knowledge-base/what-happens-during-an-execution-preconf.mdx (hunk 3 above)
Comment (offgridauthor) [reply knowledge-base/what-happens-during-an-execution-preconf.mdx:44]: added link

File: knowledge-base/what-happens-during-an-execution-preconf.mdx (hunk 2 above)
Comment (offgridauthor) [reply knowledge-base/what-happens-during-an-execution-preconf.mdx:36]: added link

File: knowledge-base/what-happens-during-an-execution-preconf.mdx (hunk 1 above)
Comment (offgridauthor) [reply knowledge-base/what-happens-during-an-execution-preconf.mdx:32]: removed this note

File: knowledge-base/what-happens-during-an-execution-preconf.mdx (hunk 4 above)
Comment (offgridauthor) [reply knowledge-base/what-happens-during-an-execution-preconf.mdx:34]: added steps

File: knowledge-base/what-happens-during-an-execution-preconf.mdx (hunk 5 above)
Comment (offgridauthor) [reply knowledge-base/what-happens-during-an-execution-preconf.mdx:40]: changed all MEV to mev

**Pull request closed without merging**